test/tools/test_component_tool.py

```diff
@@ -0,0 +1,570 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import pytest
+from typing import Dict, List, Optional, Any
+from dataclasses import dataclass
+from haystack import component
+from pydantic import BaseModel
+from haystack import Pipeline
+from haystack.components.websearch.serper_dev import SerperDevWebSearch
+from haystack.dataclasses import Document
+from haystack.dataclasses import ChatMessage, ChatRole
+from haystack.components.tools.tool_invoker import ToolInvoker
```

Comment:
imports could be sorted for better readability, for example grouping all `from haystack.components...` together